grapher/facetChart/FacetChartConstants.ts

```diff
@@ -14,9 +14,13 @@ export interface FacetSeries extends ChartSeries {
     chartTypeName?: ChartTypeName
 }
 
-export interface PlacedFacetSeries extends FacetSeries {
+export interface IntermediatePlacedFacetSeries extends FacetSeries {
     manager: ChartManager
     chartTypeName: ChartTypeName
     bounds: Bounds
     contentBounds: Bounds
 }
+
+export interface PlacedFacetSeries extends IntermediatePlacedFacetSeries {
+    chartInstance: ChartInterface
+}
```

Comment:
Do you think it's slightly better to avoid creating this additional interface and [using the `intermediateChartInstances` accessor instead](https://github.com/owid/owid-grapher/commit/bb723ea2d1baf344d60521ab127f9f7bbc2c9aeb)?

Reply:
yeah! I wasn't sure just _how intermediate_ these chart instances were.
But I'm gonna make `intermediateChartInstances` public for testing use then.

Reply:
Actually I [learned from Daniel B](https://owid.slack.com/archives/CQQUA2C2U/p1629117757012500) that you can just access them with `object["property"]` even if `object.property` is marked as private. I think that's better for tests?

> TIL: when writing tests you often want to assert the value of private properties - it turns out Typescript let's you access private properties if you use the ["propertyName"] syntax

Reply:
ah thanks, didn't know that!
just pushed another commit https://github.com/owid/owid-grapher/commit/62b1d6f019c75dc0545fe5304b4d0df7c9081338 🙌 . i should've cherry-picked your commit all along :)